lib/modules/datasource/crate/index.ts

```diff
@@ -110,7 +110,7 @@ export class CrateDatasource extends Datasource {
     result.releases = lines
       .map((version) => {
         const release: Release = {
-          version: version.vers.replace(/\+.*$/, ''),
+          version: version.vers,
```

Comment:
```suggestion
          version: version.replace(/\+.*$/, ''),
          versionOrig: version.vers,
```

maybe we should better use `gitRef` here? 🤔 
https://github.com/renovatebot/renovate/blob/937260d08e9e0242b74e4614e82aff4d05363725/lib/modules/datasource/gitlab-releases/index.ts#L55

Reply:
Hmm... but the version string here has nothing to do with git.